jerry-core/lit/lit-magic-strings.h

```diff
@@ -52,27 +52,17 @@ extern lit_utf8_size_t lit_get_magic_string_size (lit_magic_string_id_t);
 extern const lit_utf8_byte_t *lit_get_magic_string_ex_utf8 (lit_magic_string_ex_id_t);
 extern lit_utf8_size_t lit_get_magic_string_ex_size (lit_magic_string_ex_id_t);
 
-extern void lit_magic_strings_ex_set (const lit_utf8_byte_t **,
-                                      uint32_t count,
-                                      const lit_utf8_size_t *);
-
-extern  bool lit_is_utf8_string_magic (const lit_utf8_byte_t *,
-                                       lit_utf8_size_t,
-                                       lit_magic_string_id_t *);
-extern  bool lit_is_ex_utf8_string_magic (const lit_utf8_byte_t *,
-                                          lit_utf8_size_t,
-                                          lit_magic_string_ex_id_t *);
-
-extern bool lit_compare_utf8_string_and_magic_string (const lit_utf8_byte_t *,
-                                                      lit_utf8_size_t,
+extern void lit_magic_strings_ex_set (const lit_utf8_byte_t * *, uint32_t, const lit_utf8_size_t  *);
+
+extern  bool lit_is_utf8_string_magic (const lit_utf8_byte_t  *, lit_utf8_size_t, lit_magic_string_id_t  *);
+extern  bool lit_is_ex_utf8_string_magic (const lit_utf8_byte_t  *, lit_utf8_size_t, lit_magic_string_ex_id_t  *);
+
+extern bool lit_compare_utf8_string_and_magic_string (const lit_utf8_byte_t  *, lit_utf8_size_t,
```

Comment:
For the three functions above: Do we need the double spaces? and the space between `*` characters?